gui-libs/xdg-desktop-portal-hyprland/xdg-desktop-portal-hyprland-9999.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 2022 Julien Roy <julien@jroy.ca>
+# Distributed under the terms of the ISC License
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="xdg-desktop-portal backend for hyprland"
+HOMEPAGE="https://github.com/hyprwm/xdg-desktop-portal-hyprland"
+
+if [[ ${PV} == 9999 ]]; then
+	EGIT_REPO_URI="https://github.com/hyprwm/${PN}.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/hyprwm/${PN}/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv ~x86"
+fi
+
+LICENSE="BSD"
+SLOT="0"
+IUSE="elogind systemd"
+REQUIRED_USE="?? ( elogind systemd )"
+
+DEPEND="
+	>=media-video/pipewire-0.3.41:=
+	dev-libs/inih
+	dev-libs/wayland
+	media-libs/mesa
+	x11-libs/libdrm
+	|| (
+		systemd? ( >=sys-apps/systemd-237 )
+		elogind? ( >=sys-auth/elogind-237 )
+		sys-libs/basu
+	)
+	dev-qt/qtbase
+	dev-qt/qtwayland:6
+"
+# mesa is needed for gbm dep (which it hards sets to 'on')
+RDEPEND="
+	${DEPEND}
+	sys-apps/xdg-desktop-portal
+"
+BDEPEND="
+	>=dev-libs/wayland-protocols-1.24
+	virtual/pkgconfig
+"
+
+src_configure() {
+	local emesonargs=()
+
+	if use systemd; then
+		emesonargs+=(-Dsd-bus-provider=libsystemd)
+	elif use elogind; then
+		emesonargs+=(-Dsd-bus-provider=libelogind)
+	else
+		emesonargs+=(-Dsd-bus-provider=basu)
+	fi
+	meson_src_configure
+}
+
+MAKEOPTS="-C ${S}/hyprland-share-picker"
```

Comment:
It's better if you go inside the directory manually and run emake.
This variable really is only for end user settings, not for use  by developers in ebuilds. 
https://wiki.gentoo.org/wiki/MAKEOPTS

Reply:
Done